### PR TITLE
Wt new task

### PR DESCRIPTION
**Pull request by ranjan002**
TR-7:- Changing password

### CODE REVIEW

File: names.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ subl# This is a list of all the names of people working on this project
 # Format -   Full name - nickname (optional)
 Rucha Rathor
 Benjamin Chow - Ben
-Shaurav Ranjan
+Shaurav Ranjan- blue-My New Tasks
```
Comment (benachow) [inline names.md:6]: You need to make your new branch based off your master branch.  You based your WT-NEW-TASK off your color change branch.

File: centre/README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ An overview of the AEM platform can be found
       <server>
         <id>westfieldlabs-aem</id>
         <username>aem</username>
-        <password>SK8-VxW-wBu-Rb5</password>
+        <password>Z39-3xt-991-R1D</password>
```
Comment (benachow) [inline centre/README.md:35]: This is good, but there is one more password you need to change

**Review by benachow — CHANGES_REQUESTED** [review]

File: centre/README.md (hunk 2 above)
Comment (ranjan002) [reply centre/README.md:35]: ok, I will do.

Comment (benachow) [reply centre/README.md:35]: Perfect thank you

**Pull request closed without merging**